src/components/ha-selector/ha-selector-select.ts

```diff
@@ -18,23 +20,21 @@ export class HaSelectSelector extends LitElement {
   @property({ type: Boolean }) public disabled = false;
 
   protected render() {
-    return html`<ha-paper-dropdown-menu
-      .disabled=${this.disabled}
+    return html`<mwc-select
+      fixedMenuPosition
+      naturalMenuWidth
       .label=${this.label}
+      .value=${this.value}
+      .disabled=${this.disabled}
+      @closed=${stopPropagation}
+      @selected=${this._valueChanged}
```

Comment:
This doesn't seem to work

https://61fdc4fbb19bda3949bd6fca--home-assistant-gallery.netlify.app/#automation/selectors

It works currently

https://design.home-assistant.io/#automation/selectors